lib/panel-manager.js

```diff
@@ -12,18 +12,22 @@ class PanelManager {
     this.subscriptions = new CompositeDisposable()
     this.viewProviders = new Map()
     this.resizeSubscriptions = new CompositeDisposable()
+    this.initialized = false
     this.subscribeToConfig()
```

Comment:
yet your approach feels quite solid too. Just wanted to mention the actual cause...